constants/battle_anim_constants.asm

```diff
@@ -26,34 +26,34 @@ BATTLEANIM_BASE_TILE EQU 7 * 7  ; Maximum size of a pokemon picture
 
 ; BattleAnimObjects indexes (see data/battle_anims/objects.asm)
 	const_def
-	const ANIM_OBJ_00
-	const ANIM_OBJ_01
-	const ANIM_OBJ_02
-	const ANIM_OBJ_03
-	const ANIM_OBJ_04
-	const ANIM_OBJ_05
-	const ANIM_OBJ_06
-	const ANIM_OBJ_07
-	const ANIM_OBJ_08
+	const ANIM_OBJ_HIT_BIG_YFIX
+	const ANIM_OBJ_HIT_YFIX
+	const ANIM_OBJ_HIT_SMALL_YFIX
+	const ANIM_OBJ_HIT_BIG
+	const ANIM_OBJ_HIT
+	const ANIM_OBJ_HIT_SMALL
+	const ANIM_OBJ_PUNCH
+	const ANIM_OBJ_FOOT
```

Comment:
IMO do `ANIM_OBJ_KICK` to match `ANIM_OBJ_PUNCH`.

Reply:
Done